main.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/microkit/command"
+	microserver "github.com/giantswarm/microkit/server"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/viper"
+
+	"github.com/giantswarm/draughtsman-eventer/flag"
+	"github.com/giantswarm/draughtsman-eventer/server"
+	"github.com/giantswarm/draughtsman-eventer/service"
+	"github.com/giantswarm/draughtsman-eventer/service/eventer/github"
+)
+
+var (
+	description string     = "The draughtsman-eventer is an in-cluster agent that provides deployment events via a TPR/CRO."
```

Comment:
Good catch. Fixed. Should be `TPO/CRO` because the eventer fills the TPO. The TPR is created by the operator. 

IMO draughtsman is mentioned. We will have the eventer and the operator. Or what do you think should the description say?